app/code/Magento/CustomerGraphQl/Model/Resolver/CreateCustomer.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CustomerGraphQl\Model\Resolver;
+
+use Magento\CustomerGraphQl\Model\Customer\CustomerDataProvider;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Customer\Api\Data\CustomerInterfaceFactory;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Authorization\Model\UserContextInterface;
+use Magento\Customer\Api\AccountManagementInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Newsletter\Model\SubscriberFactory;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Framework\Api\DataObjectHelper;
+
+/**
+ * Create customer data resolver
+ */
+class CreateCustomer implements ResolverInterface
+{
+    /**
+     * @var CustomerDataProvider
+     */
+    private $customerDataProvider;
+
+    /**
+     * @var AccountManagementInterface
+     */
+    private $accountManagement;
+
+    /**
+     * @var CustomerInterfaceFactory
+     */
+    private $customerFactory;
+
+    /**
+     * @var DataObjectHelper
+     */
+    private $dataObjectHelper;
+    /**
+     * @var StoreManagerInterface
+     */
+    private $storeManager;
+    /**
+     * @var SubscriberFactory
+     */
+    private $subscriberFactory;
+
+    /**
+     * @param DataObjectHelper $dataObjectHelper
+     * @param CustomerInterfaceFactory $customerFactory
+     * @param AccountManagementInterface $accountManagement
+     * @param StoreManagerInterface $storeManager
+     * @param SubscriberFactory $subscriberFactory
+     * @param CustomerDataProvider $customerDataProvider
+     */
+    public function __construct(
+        DataObjectHelper $dataObjectHelper,
+        CustomerInterfaceFactory $customerFactory,
+        AccountManagementInterface $accountManagement,
+        StoreManagerInterface $storeManager,
+        SubscriberFactory $subscriberFactory,
+        CustomerDataProvider $customerDataProvider
+    ) {
+        $this->customerDataProvider = $customerDataProvider;
+        $this->accountManagement = $accountManagement;
+        $this->customerFactory = $customerFactory;
+        $this->dataObjectHelper = $dataObjectHelper;
+        $this->storeManager = $storeManager;
+        $this->subscriberFactory = $subscriberFactory;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ) {
+        if (!isset($args['input']) || !is_array($args['input']) || empty($args['input'])) {
```

Comment:
Need to have more smarty validation (based on `is_required` values of Customer EAV attributes)

Reply:
`Magento\CustomerGraphQl\Model\Resolver\UpdateCustomer` does not contain any validation related to settings in admin panel (there is only one rule `need to provide password if you change email`)